### PR TITLE
Adds [orbit_settings] template to the extension.toml, adds 'dependencies' section to README.md

### PR DESCRIPTION
**Pull request by hhansen-bdai**
Added [orbit_settings] TODOs to the extension.toml, added section to  README.md about dependency installation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +129,16 @@ To enable your extension, follow these steps:
...
     - Toggle it to enable your extension.
 
+
+### Installing Dependencies
+
+To ensure that your program works as expected, please add your extensions's dependencies to the appropriate configuration file. Below are links for how to specify extension dependencies on ``IsaacSim`` and ``Orbit`` extensions, ``pip`` packages, ``apt`` packages, and [rosdep packages](https://docs.ros.org/en/humble/Tutorials/Intermediate/Rosdep.html).
+
+- [Extensions](https://docs.omniverse.nvidia.com/kit/docs/kit-manual/latest/guide/extensions_advanced.html#dependencies-section)
+- [pip packages](https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/#install-requires)
+- [apt packages](TODO: Put link to Orbit docs when merged)
```
Comment (hhansen-bdai) [inline README.md:139]:
```suggestion
- [apt packages](https://isaac-orbit.github.io/orbit/source/setup/developer.html#extension-dependency-management)
```

File: README.md [hunk 2]
```diff
@@ -129,6 +129,16 @@ To enable your extension, follow these steps:
...
+- [rosdep packages](TODO: Put link to Orbit docs when merged)
```
Comment (hhansen-bdai) [inline README.md:140]:
```suggestion
- [rosdep packages](https://isaac-orbit.github.io/orbit/source/setup/developer.html#extension-dependency-management)
```

**Review by nburger-bdai — APPROVED** [review]
LGTM!

**Comment by Mayankm96** [comment]
PLEASE make sure the PR template is respected and the title of the MR is concise.

MR title is not equal to PR description :/

**Pull request merged**